src/librustdoc/passes/collapse_docs.rs

```diff
@@ -1,72 +1,14 @@
-use crate::clean::{self, DocFragment, DocFragmentKind, Item};
+use crate::clean;
 use crate::core::DocContext;
-use crate::fold;
-use crate::fold::DocFolder;
 use crate::passes::Pass;
 
-use std::mem::take;
-
 crate const COLLAPSE_DOCS: Pass = Pass {
     name: "collapse-docs",
     run: collapse_docs,
     description: "concatenates all document attributes into one document attribute",
 };
 
-crate fn collapse_docs(krate: clean::Crate, _: &DocContext<'_>) -> clean::Crate {
-    let mut krate = Collapser.fold_crate(krate);
+crate fn collapse_docs(mut krate: clean::Crate, _: &DocContext<'_>) -> clean::Crate {
     krate.collapsed = true;
     krate
 }
```

Comment:
This seems kind of silly - can we just remove the pass altogether?

Reply:
This is mostly a POC. The goal is to first check the perf outcome. If it's the right direction, then we can investigate this further. For now it was simply easier to do it this way.

Reply:
Can you remove this now you've done the perf run?

Reply:
This is still waiting on a response.

Reply:
Removed the whole pass.